src/pyodide.js

```diff
@@ -330,6 +331,41 @@ var languagePluginLoader = new Promise((resolve, reject) => {
   Module.preloadedWasm = {};
   let isFirefox = navigator.userAgent.toLowerCase().indexOf('firefox') > -1;
 
+  Module.runPython = code => Module.py_pyodide.eval_code(code, Module.globals);
+
+  // clang-format off
+  Module.loadPackagesForCode = async function(code, messageCallback, errorCallback) {
```

Comment:
Maybe,
```suggestion
  Module.loadPackagesFromImports = async function(code, messageCallback, errorCallback) {
```
to be more consistent with `pyodide.find_imports`? Also it's not really loading packages used in the code, only imports matter.